src/invidious/migrations/0007_create_playlists_table.cr

```diff
@@ -0,0 +1,47 @@
+module Invidious::Migrations
+  class CreatePlaylistsTable < Migration
+    version 7
+
+    def up(conn : DB::Connection)
+      conn.exec <<-SQL
+      DO
+      $$
+      BEGIN
+        IF NOT EXISTS (SELECT *
+                        FROM pg_type typ
+                        INNER JOIN pg_namespace nsp ON nsp.oid = typ.typnamespace
+                        WHERE nsp.nspname = 'public'
+                              AND typ.typname = 'privacy') THEN
+          CREATE TYPE public.privacy AS ENUM
+            (
+                'Public',
+                'Unlisted',
+                'Private'
+            );
+        END IF;
+      END;
+      $$
+      LANGUAGE plpgsql;
+      SQL
```

Comment:
This is the only sql that was changed to allow for it to be run if the privacy type already exists. Got the sql to do it from https://stackoverflow.com/questions/56647514/create-a-type-if-not-exist-postgresql